examples/rag/finetune_rag.py

```diff
@@ -509,4 +599,29 @@ def main(args=None, model=None) -> GenerativeQAModule:
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser()
+    parser = pl.Trainer.add_argparse_args(parser)
+    parser = GenerativeQAModule.add_model_specific_args(parser, os.getcwd())
+    parser = GenerativeQAModule.add_retriever_specific_args(parser)
+
+    # Pytorch Lightning Profiler
+    parser.add_argument(
+        "--profile",
+        action="store_true",
+        help="If True, use pytorch_lightning.profiler.AdvancedProfiler to " "profile the Trainer.",
+    )
+
+    # Ray cluster address
+    parser.add_argument(
```

Comment:
Maybe group the ray-specific args (`ray-address` and `num_retrieval_workers `) with a function `add_ray_specific_args` ?

Reply:
Done!